src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -271,140 +290,110 @@
       (Thread/sleep 10)
       (recur curr-time))))
 
-(def supported-cli-options
-  [["-c" "--config CONFIG" "Path to config or conf.d directory (required)"]
-   ["-F" "--facts FACTS" "Path to a directory containing sample JSON facts (files must end with .json)"]
-   ["-C" "--catalogs CATALOGS" "Path to a directory containing sample JSON catalogs (files must end with .json)"]
-   ["-R" "--reports REPORTS" "Path to a directory containing sample JSON reports (files must end with .json)"]
-   ["-A" "--archive ARCHIVE" "Path to a PuppetDB export tarball. Incompatible with -C, -F, -R, or -D"]
-   ["-i" "--runinterval RUNINTERVAL" "What runinterval (in minutes) to use during simulation"]
-   ["-n" "--numhosts NUMHOSTS" "How many hosts to use during simulation"]
-   ["-r" "--rand-perc RANDPERC" "What percentage of submitted catalogs are tweaked (int between 0 and 100)"]
-   ["-N" "--nummsgs NUMMSGS" "Number of commands and/or reports to send for each host"]])
-
-(def required-cli-options
-  [:config :numhosts])
-
-(defn activate-logging!
-  [options]
-  (-> (:config options)
-      (get-in [:global :logging-config])
-      logutils/configure-logging!))
-
 (defn validate-options
   [options action-on-error-fn]
-  (activate-logging! options)
   (cond
     (and (contains? options :runinterval)
          (contains? options :nummsgs))
-    (do
-      (log/error "Error: -N/--nummsgs runs immediately and is not compatable with -i/--runinterval")
-      (action-on-error-fn))
-
-    (not-any? #{:nummsgs :runinterval} (keys options))
-    (do
-      (log/error "Error: Either -N/--nummsgs or -i/--runinterval is required.")
-      (action-on-error-fn))
+    (utils/throw+-cli-error! "Error: -N/--nummsgs runs immediately and is not compatable with -i/--runinterval")
+    
+    (and (not (contains? options :runinterval))
+         (not (contains? options :nummsgs)))
```

Comment:
Oops yeah that wasn't meant to be in this commit anyways, but there's actually a `kitchensink` function called `missing?` that does this a little better